ProgrammingTask/server.py

```diff
@@ -15,31 +16,68 @@
 s.listen(3) #3 clients can connect
 print("Waiting for a connection, Server Started")
 
-def threaded_client(conn, addr):
-    conn.send(str.encode("Connected"))
-    reply = ""
+connected = set()
+games = {}
+idCount = 0
+
+def threaded_client(conn, playerId, gameId):
+    global idCount
```

Comment:
We are using classes to encapsulate state in other modules, i think this could be easily  converted to class too, so that global variables are not needed. The code would be more consistent that way.

So we would have something like:
```python
class RPSServer:
    def __init__(self, ...):
        self.id_count = 0
...
```